nntrainer/graph/network_graph.cpp

```diff
@@ -515,10 +515,14 @@ sharedConstTensors NetworkGraph::forwarding(bool training) {
 }
 
 std::vector<TensorDim> NetworkGraph::getInputDimension() {
+  NNTR_THROW_IF(num_node == 0, std::invalid_argument)
```

Comment:
Please add a `TODO` here that this works only after graph compilation.
This will be fixed to work always with #1020.

Reply:
Added

Reply:
Added